src/view_neume.cpp

```diff
@@ -274,7 +274,9 @@ void View::DrawNc(DeviceContext *dc, LayerElement *element, Layer *layer, Staff
     }
 
     // Draw the children
-    this->DrawLayerChildren(dc, nc, layer, staff, measure);
+    if (!nc->FindDescendantByType(LIQUESCENT)) {

```

Comment:
Normally we always render child elements individually. So if the encoding is:
```xml
<nc>
  <liquescent/>
</nc>
```
We expect the liquescent to be drawn in a call to `DrawLayerChildren` so the SVG will be
```xml
<g class="nc">
  <g class="liquescent"/>
</g>
```
This is important if you have editorial markup:
```xml
<nc>
  <add>
     <liquescent/>
   </add>
</nc>
```
Which then becomes:
```xml
<g class="nc">
  <g class="add">
    <g class="liquescent"/>
  <g/>
</g>
```
Which allows (for example) to highlight what has been added. This would be missing with your implementation because `<liquescent>` drawing is happening in `nc` drawing.